packages/rs-drive-abci/src/execution/fee_pools/fee_distribution.rs

```diff
@@ -874,7 +874,7 @@ mod tests {
             assert!(matches!(
                 proposer_payouts,
                 Some(ProposersPayouts {
-                    proposers_paid_count: proposers_count,
+                    proposers_paid_count: _proposers_count,
```

Comment:
You need to add a guard to check the value or add `..` instead if you don't care about it